src/style.css

```diff
@@ -0,0 +1,21 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --text-color-main: #060b35;
+  --text-color-secondary: #616070;
+  --bg-color-blue: #00acdc;
+  --bg-color-white: #fff;
+}
```

Comment:
You could create a separate file for declaring variables. 
Also, I would put the other styles here in another `.css` file and use this file only to import the other style sheets.